src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol_Bases.cs

```diff
@@ -319,6 +319,12 @@ internal override ImmutableArray<NamedTypeSymbol> GetDeclaredInterfaces(ConsList
                 }
             }
 
+            if (declaration.Kind == DeclarationKind.Record)
+            {
+                var type = DeclaringCompilation.GetWellKnownType(WellKnownType.System_IEquatable_T);
```

Comment:
>DeclaringCompilation.GetWellKnownType(WellKnownType.System_IEquatable_T); [](start = 27, length = 73)

Please properly handle missing and otherwise bad types.

Reply:
Use-site errors are already reported for interfaces. See `RecordTests.IEquatableT_08`.